.travis.yml

```diff
@@ -1,10 +1,25 @@
-language: node_js
-node_js:
-  - "8"
+language: python
 cache:
   directories:
     - "node_modules"
 install:
-  - npm install
+  - "if [[ \"$GROUP\" == js ]] ; then npm -v ; fi"
+  - "if [[ \"$GROUP\" == js ]] ; then npm install ; fi"
+  - "if [[ \"$GROUP\" == python-linting ]] ; then pip install black flake8 ; fi"
 script:
-  - npm run test
+  - "if [[ \"$GROUP\" == js ]] ; then npm run test ; fi"
+  - "if [[ \"$GROUP\" == python-linting ]] ; then black . --check ; fi"
+  - "if [[ \"$GROUP\" == python-linting ]] ; then flake8 ; fi"
```

Comment:
Here I think we want `flake8 --ignore E203, W503`.  These are, respectively, "whitespace before ':'" and "line break before binary operator" warnings, which `flake8` will complain about, but which aren't PEP8 compliant and `black` does not adhere to. There's a bit of discussion about it in the `black` [README](https://github.com/ambv/black).

Reply:
Good call. I'll put these in `setup.cfg` (it's better to have these in configuration, rather than just in CI, so that tools that integrate with editors can use them as well.

Reply:
Sounds great!